cli/Application/Command/Make/Langtemplates.php

```diff
@@ -73,7 +73,7 @@ public function execute()
 		$this->getApplication()->outputTitle('Make Language templates');
 
 		ExtensionHelper::addDomainPath('Core', JPATH_ROOT . '/src');
-		ExtensionHelper::addDomainPath('CoreJS', JPATH_ROOT . '/www/media');
+		ExtensionHelper::addDomainPath('CoreJS', JPATH_ROOT . '/www/media/js');
 		ExtensionHelper::addDomainPath('Template', JPATH_ROOT . '/cache/twig');
```

Comment:
I knew it was spitting errors but I didn't know enough about how it was all working to fix it, the generated files are still right, and it wasn't causing it to break on the live server, so I left it alone.

I guess I can change the cache path on the live server to make sure things don't conflict.  It was one of those things where I saw it and thought "wait a second, 2 things using the same cache for 2 purposes?".